src/views/Projects/index.jsx

```diff
@@ -0,0 +1,147 @@
+import { Component } from 'react';
+import { withStyles } from 'material-ui/styles';
+import ReactMarkdown from 'react-markdown';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Typography from 'material-ui/Typography';
+import TextField from '@material-ui/core/TextField';
+import projects from '../../data/loader';
+import CustomizedCard from '../../components/CustomizedCard';
+
+const styles = theme => ({
+  root: {
+    flexGrow: 1,
+    zIndex: 1,
+    overflow: 'hidden',
+    position: 'relative',
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+  },
+  mainCard: {
+    backgroundColor: theme.palette.primary.dark,
+    margin: theme.spacing.unit * 3,
+  },
+  text: {
+    color: theme.palette.secondary.light,
+    margin: '1px',
+  },
+  grid: {
+    padding: '0 15px !important',
+    maxWidth: '100%',
+  },
+  content: {
+    flexGrow: 1,
+    backgroundColor: theme.palette.background,
+    minWidth: 0,
+  },
+  container: {
+    paddingRight: '15px',
+    paddingLeft: '15px',
+    marginRight: 'auto',
+    marginLeft: 'auto',
+  },
+  flex1: {
+    flex: 1,
+  },
+  columnFlex: {
+    display: 'flex',
+    flexDirection: 'column',
+  },
+});
+
+class Projects extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projects,
+      search: '',
+    };
+  }
+
+  handleTextInputChange = event => {
+    this.setState({ search: event.target.value });
+  };
+  render() {
+    const { classes } = this.props;
+    const { projects, search } = this.state;
+    const selectedProjects = Object.keys(projects)
+      .filter(fileName => {
+        const found =
+          fileName.toLowerCase().indexOf(search.toLowerCase()) !== -1 ||
+          projects[fileName].name
+            .toLowerCase()
+            .indexOf(search.toLowerCase()) !== -1;
+
+        return found;
+      })
+      .reduce(
+        (prev, key) => ({
+          ...prev,
+          [key]: projects[key],
+        }),
+        {}
+      );
+
+    return (
+      <div className={classes.root}>
+        <header>
+          <Grid
+            container
+            spacing={16}
+            direction="column"
+            justify="center"
+            alignItems="center">
+            <Grid item xs={12}>
+              <div className={classes.columnFlex}>
+                <div className={classes.flex1} />
+                <div>
+                  <Typography variant="display4" align="center">
+                    Bug Issue UI
```

Comment:
I think you can just use Bug Issue.

Reply:
I don't think you need of all those `div`s wrappers. See https://github.com/mozilla-frontend-infra/bug-issue-ui/pull/12#discussion_r189601958 for how you can simply put it under a `Fragment`. Since the title will always be on its own line, we also don't need it to be wrapped inside a `Grid`.